cli/config.go

```diff
@@ -94,6 +96,9 @@ var configDefaults = map[string]any{
 
 // defaultConfig returns a new config with default values.
 func defaultConfig() *viper.Viper {
+	// load environment variables from .env file if one exists
+	_ = godotenv.Load()
```

Comment:
todo: viper has built in support for envfiles. We should be able to remove this dependency

Reply:
I think viper can read from only one config file at a time. We already have the defra config file that is used with viper.

Reply:
ahh okay that seems like an odd choice on their part